avm/res/kubernetes-configuration/flux-configuration/tests/e2e/max/main.test.bicep

```diff
@@ -74,6 +74,12 @@ module testDeployment '../../../main.bicep' = [for iteration in [ 'init', 'idem'
         prune: true
         syncIntervalInSeconds: 300
         timeoutInSeconds: 300
+        postBuild: {
```

Comment:
Hey @aadev1, can you see anything in this PR which may be incorrect? As you can see in the pipeline badge, I tried to deploy the solution as per the changes you suggested. For some reason though, 2 attempts of deployments timed out after 6h.

Reply:
Hi @AlexanderSehr,  I've had a look through the PR and I can't spot anything wrong.

Reply:
Updated it to an API version in between the latest and the one before which works